tests/unit_tests/test_material.py

```diff
@@ -58,6 +58,96 @@ def test_elements_by_name():
     assert a._nuclides == b._nuclides
     assert b._nuclides == c._nuclides
 
+def test_adding_elements_by_formula():
```

Comment:
PEP8: two blank lines between functions at top level
```suggestion

def test_add_elements_by_formula():
```